src/main/java6/net/finmath/time/ScheduleGenerator.java

```diff
@@ -127,8 +126,8 @@ private ScheduleGenerator() {
 	 * t = 0 corresponds to the reference date.
 	 * 
 	 * @param referenceDate The date which is used in the schedule to internally convert dates to doubles, i.e., the date where t=0.
-	 * @param startDate The start date of the first period.
-	 * @param maturity The end date of the first period.
+	 * @param startDate The start date of the first period (this may/should be an unadjusted date).
```

Comment:
But then I would also like to demand this and throw an exception otherwise. If there is a legit startDate which is not a busDay then my feeling tells me it should be unadjusted (in case you have a short period at the beginning). What do you think?